trappy/thermal.py

```diff
@@ -88,7 +88,7 @@ def plot_temperature(self, control_temperature=None, title="", width=None,
             setup_plot = True
 
         temp_label = normalize_title("Temperature", legend_label)
-        (thermal_dfr["temp"] / 1000).plot(ax=ax, label=temp_label)
+        (thermal_dfr["temp"] / 1000).plot(ax=ax, label=temp_label, drawstyle="steps-post")
```

Comment:
Not sure about this, if you really want to see a plot of the temperature, then I think linear interpolation makes perfect sense. If you're looking at thermal_temperature to find out what input a kernel component was getting then I suppose steps-post makes sense, but I'd probably be happy if I had to set that manually when requesting the plot - can you make it a param the caller can set?